cli/cmd/root.go

```diff
@@ -80,7 +81,7 @@ func init() {
 	RootCmd.AddCommand(newCmdInject())
 	RootCmd.AddCommand(newCmdInstall())
 	RootCmd.AddCommand(newCmdLogs())
-	RootCmd.AddCommand(newCmdProfile())
+	RootCmd.AddCommand(profile.NewCmdProfile(controlPlaneNamespace))
```

Comment:
for consistency with the rest of the cmd's, how about leaving `cmd/profile.go`, and then having that pass `controlPlaneNamespace` to the `cmd/profile` package?

or, going a step further, maybe it makes sense to move everything under `cmd/profile` down into `pkg/profiles`? this would make `renderOpenAPI` and `renderProto` consistent with `RenderProfileTemplate`

related, you could avoid making `profileOptions` public by a slight change to the render functions:
```go
func RenderOpenAPI(input, name, namespace string, controlPlaneNamespace string, w io.Writer) error {}
func RenderProto(input, name, namespace string, controlPlaneNamespace string, w io.Writer) error {}
```